gpMgmt/test/behave/mgmt_utils/gpactivatestandby.feature

```diff
@@ -76,6 +76,27 @@ Feature: gpactivatestandby
           And the tablespace is valid on the standby coordinator
           And clean up and revert back to original coordinator
 
+    Scenario: gprecoverseg recoveries failed segment when standby works instead of coordinator
+        Given the database is running
+          And all the segments are running
+          And the segments are synchronized
+          And the standby is not initialized
+         When the user initializes a standby on the host "sdw3" and port 7001
```

Comment:
why not `running gpinitstandby on host "{coordinator}" to create a standby on host "{standby}`?